client/src/service/resolve.ts

```diff
@@ -16,12 +16,15 @@ export const resolve = async (identifier: string): Promise<DIDDocument> => {
     connection,
     identifierToPubkey(identifier)
   );
-  // TODO map solidData to DIDDocument
-  return {
-    '@context': 'https://w3id.org/did/v1',
-    id: identifier,
-    // @ts-ignore
-    temp: solidData,
-    publicKey: [],
-  };
+  if (solidData !== null) {
```

Comment:
probably we should just error out here, or return an empty response, right? Np real value in keeping the dummy value

Reply:
Throwing is probably best -- I didn't want to change the signature

Reply:
ok I'll deal with it

Reply:
aah I see you did 👍